src/main/resources/org/jenkinsci/plugins/oic/Messages.properties

```diff
@@ -3,19 +3,21 @@ OicLogoutAction.OicLogout = Oic Logout
 OicSecurityRealm.DisplayName = Login with Openid Connect
 OicSecurityRealm.ClientIdRequired = Client id is required.
 OicSecurityRealm.ClientSecretRequired = Client secret is required.
-OicSecurityRealm.URLNotAOpenIdEnpoint = URL does seem to describe OpenID Connect endpoints
+OicSecurityRealm.URLNotAOpenIdEnpoint = URL does not seem to describe OpenID Connect endpoints
 OicSecurityRealm.NotAValidURL = Not a valid url.
 OicSecurityRealm.CouldNotRetreiveWellKnownConfig = Could not retrieve well-known config {0,number,integer} {1}
 OicSecurityRealm.CouldNotParseResponse = Could not parse response
 OicSecurityRealm.ErrorRetreivingWellKnownConfig = Error when retrieving well-known config
 OicSecurityRealm.TokenServerURLKeyRequired = Token Server Url Key is required.
 OicSecurityRealm.TokenAuthMethodRequired = Token auth method is required.
 OicSecurityRealm.UsingDefaultUsername = Using ''sub''.
-OicSecurityRealm.UsingDefaultScopes = Using ''openid email''.
-OicSecurityRealm.RUSureOpenIdNotInScope = Are you sure you don''t want to include ''openid'' as an scope?
+OicSecurityRealm.RUSureOpenIdNotInScope = Are you sure you don''t want to include ''openid'' as a scope?
+OicSecurityRealm.ScopesRequired = Scopes is required.
```

Comment:
sounds like bad English, but "scopes" is the field, not a plural of "scope" so the field is singular.
Probably should be reworded so that it has something other than just "the field is required"